data/assets/examples/screenspacerenderable/screenspacetimevaryingimageonline/data/example.json

```
@@ -0,0 +1,16 @@
+{
+ "files": [
+  {
+   "timestamp": "2024-05-10 15:00:00.0",
+   "url": "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg"
+  },
+  {
+   "timestamp": "2024-05-10 15:10:00.0",
+   "url": "https://data.openspaceproject.com/examples/renderableplaneimageonline_2.jpg"
+  },
+  {
+   "timestamp": "2024-05-10 15:20:00.0",
+   "url": "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg"
```

```suggestion
   "timestamp": "2024-05-10 15:00:00.000",
   "url": "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg"
  },
  {
   "timestamp": "2024-05-10 15:10:00.000",
   "url": "https://data.openspaceproject.com/examples/renderableplaneimageonline_2.jpg"
  },
  {
   "timestamp": "2024-05-10 15:20:00.000",
   "url": "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg"
```

Didn't notice them before, sorry